src/components/UserInfo.js

```diff
@@ -16,4 +17,8 @@ export default class UserInfo {
     this.userName.textContent = name;
     this.userDescription.textContent = description;
   }
+
+  setAvatar(avatar) {
```

Comment:
Great that you have a method for setting the avatar, since it's controlled by a different form